composer.json

```diff
@@ -3,6 +3,7 @@
     "description": "Extends Shopware 6 transaction mails with a Schema.org conform HTML content",
     "type": "shopware-platform-plugin",
     "license": "MIT",
+    "version": "1.0.1",
```

Comment:
I'm not sure about the version. In theory it's a breaking change, but afaik no one uses this plugin at the moment.